src/Swarm/Language/Capability.hs

```diff
@@ -193,6 +193,7 @@ constCaps = \case
   Exp -> Just CArith
   Whoami -> Just CWhoami
   Self -> Just CWhoami
+  Swap -> Just CAtomic
```

Comment:
@byorgey this seems wrong, but I didn't want to add it to the already overloaded grabber. :confused: What do you think?

Whichever I choose, the device description needs to be updated too.

Reply:
Maybe make a new device, `grabber` + `rubber band` yields a... um... `rubber grabber`?  And a `rubber grabber` is an upgraded (#480) `grabber` that additionally provides `swap` capability?

Reply:
I made a fast grabber :+1: It looks like two grabbers (`≪`) and requires two grabbers and a rubber band to build. :grinning: